packages/client/src/components/measurements/MeasurementStatisticsBelowChart.re

```diff
@@ -0,0 +1,131 @@
+[@react.component]  
+let make =  
+    (
+        ~cdf: ForetoldComponents.Types.Dist.t, 
+        ~measurable: Types.measurable
+    ) => {
+    
+    module Calculate = {
```

Comment:
The formatting of this document is kind of weird. I recommend using something like reason-vscode to auto-format on save.